exec/plugin-commands-script-runners/src/run.ts

```diff
@@ -159,7 +171,11 @@ export async function handler (
       : undefined
     return printProjectCommands(manifest, rootManifest ?? undefined)
   }
-  if (scriptName !== 'start' && !manifest.scripts?.[scriptName]) {
+
+  const multiScriptSelectorRegExp = buildRegExpFromCommand(scriptName)
```

Comment:
it won't always make a regex, so probably better to name it
```suggestion
  const multiScriptSelectorRegExp = tryBuildRegExpFromCommand(scriptName)
```

Reply:
the variable can probably be names just `scriptsSelector`

Reply:
looks good. Thanks! [e140531](https://github.com/pnpm/pnpm/pull/5871/commits/e14053198946d5dcf95f758916337f5fe39c45cb)